RELEASE.md

```diff
@@ -0,0 +1,11 @@
+# Sceptre release process
+
+1. Bump version in `./sceptre/__init__.py` & `./setup.cfg`
+2. Update Changelog with details from
+   `git log --no-merges --format='%<(20)%an' v1.0.0..HEAD | sort | uniq`
+   where the version number is the git tag of the last release.
+3. Create a PR for above changes
+4. `git tag -as vX.Y.Z`
```

Comment:
@zxiiro Is this done AFTER the squash-merge to master or before?

Reply:
It should be done after merge so I guess we should have a step for merging and then doing a final `git pull`. Let me add that.